ui_utils/interaction_manager/src/lib_components/Keyboard.js

```diff
@@ -0,0 +1,144 @@
+import { getKeys, getKeysIndex } from '../utils/global-object-utility-functions';
+import mappings from '../utils/keyboard-mappings';
+import Actions from './Actions';
+/**
+ * Keyboard class that handles all keyboard interactions
+ */
+class Keyboard {
+    /* eslint-disable-next-line require-jsdoc */
+    constructor() {
+        this.mappings = mappings;
+        this.eventListenerAttached = false;
+
+        this.keysPressed = new Set();
+
+        this.onKeyDown = this.onKeyDown.bind(this);
+        this.onKeyUp = this.onKeyUp.bind(this);
+    }
+
+    /**
+     * @param {Object} options
+     * @param {string[]} options.keys Array of keys you want to use, allows only combination of modifier and regular keys
+     * @param {function | string} options.callback Function or action to be executed on the key combination
+     * @param {('press'|'hold'|'lift')} options.type Type of key action you want to use.

```

Comment:
```suggestion
     * @param {string} options.type Type of key action you want to use.

```

Reply:
We discussed this and decided to leave it like that as it's more useful in the editor: https://coherentlabs.slack.com/archives/C7CEWG1CL/p1650277958583549